pkg/sql/opt/xform/testdata/placeholder-fast-path/scan

```diff
@@ -300,3 +300,30 @@ placeholder-fast-path
 SELECT * FROM xyz WHERE x = $1 AND y = $2
 ----
 no fast path
+
+# Regresssion test for #81315. Do not use the placeholder fast path
```

Comment:
(if you're patching something, there's a typo on "Regression" here whilst you're at it)